rpm/pacemaker.spec.in

```diff
@@ -55,11 +55,11 @@
 # Define conditionals so that "rpmbuild --with <feature>" and
 # "rpmbuild --without <feature>" can enable and disable specific features
 
-## Add option to enable support for stonith/external fencing agents
+## Add option for Linux-HA (stonith/external) fencing agent support
 %if 0%{?suse_version} > 0
-%bcond_without stonithd
+%bcond_without linuxha
```

Comment:
Some part of this commit seems to have broken OpenSUSE builds:
```
15:03:16  make WITH="--with linuxha" rpm
15:03:16  make: *** No rule to make target 'linuxha"'.  Stop.
```

though it looks like a syntax issue (notice that the second double quote is included in the bogus make target name), so I'm not sure how `with stonithd` ever worked either

Reply:
Actually it's a companion PR in ci-tools that enabled `--with linuxha` for suse builds (we haven't been building that code in CI so far). That had two problems, ci-tools is dropping the quotes somewhere along the line so make thinks `linuxha"` is a make target, and I forgot to add the default `--without doc` to the suse string and it's failing on a doc dependency.